frameworks/nestjs/prisma/schema.prisma

```diff
@@ -4,7 +4,7 @@ generator client {
 
 datasource db {
     provider = "postgresql"
-    url      = env("FRAMEWORK_NESTJS_PG_URL")
+    url      = env("DATABASE_URL")
```

Comment:
avoids a clash with the new default logic that runs when the env var is `DATABASE_URL`